README.md

```diff
@@ -6,7 +6,7 @@ This project is a docker image for developing Node.js and Python web projects. I
 ## Image Features
 
 - Based on ubuntu:focal
-- Python 3.8
+- Python 3.10
```

Comment:
Same here, I believe using `ubuntu:noble` provides a more recent version of python3.